docs/docs/testing-your-assistant.mdx

```diff
@@ -377,6 +377,26 @@ The test script will also save a confusion matrix to a file called
 matrix shows how often the action was correctly predicted and how often an
 incorrect action was predicted instead.
 
+### Interpreting the generated warnings
+
+The test script will also generate a warnings file called `results/stories_with_warnings.yml`.
+This file contains all test stories for which [`action_unlikely_intent`](./default-actions.mdx#action_unlikely_intent)
+was predicted at any conversation turn but all actions from the original story were predicted correctly.
+However, if a test story originally consisted of `action_unlikely_intent` but wasn't predicted by
+the ensemble of policies, then the corresponding story will end up in `results/failed_test_stories.yml` as
+a failed story.
+
+The stories are sorted by the severity of `action_unlikely_intent`'s prediction.
+This severity is calculated by `IntentTEDPolicy` itself at prediction time.
+Higher the severity, the more unlikely is the intent and hence reviewing that particular
+conversation path becomes more critical.
+
+It should be noted that `action_unlikely_intent` is predicted by
+[`IntentTEDPolicy`](./policies.mdx#intenttedpolicy) which employs a machine learning based model
+under the hood and hence can result in multiple false warnings as well. Users are encouraged to go through the
```

Comment:
remove "multiple"

Reply:
"You should" or "We recommend you go through..." instead of "User are encouraged to"